website/static/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
unless this gets copied over to root of `gh-pages` branch automatically? it will need to be a separate pr to `gh-pages` branch

Reply:
`yarn build` takes care of it.